src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -46,8 +46,8 @@ def _polish_bad_errors(ex):
                               "which has never contained a Windows worker, and vice versa. " +
                               "Please use a new resource group. Original error:" + detail)
             ex = CLIError(detail)
-        except Exception:  # pylint: disable=broad-except
-            pass
+        except json.JSONDecodeError:
```

Comment:
My bad: my example was not completed. Please change it to
`except (json.JSONDecodeError, ValueError):`
The `ValueError` was thrown in Python 2.

Reply:
You know what. This turns into a bad idea because of `json.JSONDecoderError` doesn't exist in Python2. 

Please revert this change and use your original form. It is my mad.